pkg/processor/trigger/partitioned/eventhub/partition.go

```diff
@@ -89,10 +89,13 @@ func (p *partition) Read() error {
 		}
 
 		// Accept message
-		msg.Accept()
+		err = msg.Accept()
+		if err != nil {
+			return errors.Wrap(err, "Error Accepting message from AMQP")
+		}
 
 		// set event data
-		p.event.body = msg.Data
+		p.event.body = msg.Data[0]
```

Comment:
Accept multi-array input (entire `msg.Data` as is must pass to the function handler)

Reply:
impossible basically, maybe can happen next pr
reason: requires wide changes on event interface on nuclio-sdk-go as well